postgres/10-alpine/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM postgres:10-alpine
 
 LABEL maintainer="Debezium Community"
-ARG USE_POSTGIS=true
+ARG USE_POSTGIS=false
```

Comment:
Why is this changing?

Reply:
@gunnarmorling Sorry, I forgot to link the issue - https://issues.jboss.org/browse/DBZ-1307

Reply:
Ok, thanks. So are you planning to re-enable this in a follow-up as per "We need to make the package from source now to be able to build PostGIS"?

Reply:
@gunnarmorling Yes, just it has lower priority now. I am no Alpine expert so we can expect some lerning curve.

Reply:
+1 Can you only file a JIRA issue so it's tracked. Thanks!

Reply:
@gunnarmorling I am now a little bit confused - Jira issue for what? I've already logged one

Reply:
Ah, so I was confused, too :) I thought 1307 was to document the disablement of PostGIS under Alpine, whereas it's meant for re-enabling this. So this warrants a mention in the release notes then.